scripts/docs/build-ckeditor5.mjs

```diff
@@ -10,20 +12,31 @@ import upath from 'upath';
 import fs from 'fs-extra';
 import { CKEDITOR5_ROOT_PATH, CKEDITOR5_COMMERCIAL_PATH } from '../constants.mjs';
 import generateCKEditor5DocsBuild from './generate-ckeditor5-docs-build.mjs';
+import parseArguments from '../docs/parse-arguments.mjs';
+
+buildSources()
+	.catch( () => {
+		process.exitCode = 1;
+	} );
+
+async function buildSources() {
```

Comment:
Filename has been changed. Is it worth to change the function name as well?

Reply:
Since it's hard to me to find a proper name, I decided to use `buildCKEditorPackages()` and `build-ckeditor-packages.mjs`. Done in 0d94db1.